lib/matplotlib/cbook/deprecation.py

```diff
@@ -249,11 +249,15 @@ def wrapper(*args, **kwargs):
         old_doc = inspect.cleandoc(old_doc or '').strip('\n')
 
         message = message.strip()
-        new_doc = ('[*Deprecated*] {old_doc}\n'
-                   '\n'
-                   '.. deprecated:: {since}\n'
-                   '   {message}'
-                   .format(since=since, message=message, old_doc=old_doc))
+        # Add "Deprecated" to top of docstring
```

Comment:
Can't we simply

~~~
optional_notes_section = '\nNotes\n-----\n'

new_doc = f"""[*Deprecated*] {old_doc}
{optional_notes_section}
.. deprecated:: {since}
   {message}
"""

Reply:
Sure, feel free to push to my branch...

Reply:
Looks much better, thanks a lot!